tests/api/test_itemtypes.py

```diff
@@ -263,20 +263,29 @@ def test_get_collection_items(config, api_):
     rsp_headers, code, response = get_collection_items(api_, req, 'obs')
     features = json.loads(response)
 
+    assert len(features['features']) == 0
+
+    req = mock_api_request({
+        'offset': '1',
+        'limit': '1'
+    })
+    rsp_headers, code, response = get_collection_items(api_, req, 'obs')
+    features = json.loads(response)
+
     assert len(features['features']) == 1
 
     links = features['links']
     assert len(links) == 6
-    assert '/collections/obs/items?f=json&limit=1&bbox=-180,90,180,90' in \
+    assert '/collections/obs/items?f=json&limit=1' in \
```

Comment:
If the bbox tests are removed from here, does it mean we are going to create separate tests for each features (and records) provider?

Reply:
I removed this test because of previous behaviour of WxS (if property is null, it doesn't evaluate to true n a query/match).  Looking at OAFeat [Req 24C](https://docs.ogc.org/is/17-069r4/17-069r4.html#_parameter_bbox), I've updated this to true, hence restorng the existing behaviour in `tests/api/test_itemtypes.py` (the remaining changes being updating the miny value in the request and the resulting links).